vertx/echo/EchoClient.java

```diff
@@ -1,25 +1,7 @@
-/*
```

Comment:
The original was taken from the vertx repo and the examples do not contain copyrights.